modules/mfa/www/prompt-for-mfa.php

```diff
@@ -124,7 +125,7 @@
 $t->data['stateId'] = $stateId;
 $t->data['supportsWebAuthn'] = LoginBrowser::supportsWebAuthn($userAgent);
 $browserJsHash = md5_file(__DIR__ . '/simplewebauthn/browser.js');
-$t->data['browserJsPath'] = 'simplewebauthn/browser.js?v=' . $browserJsHash;
+$t->data['browserJsPath'] = '/module.php/mfa/simplewebauthn/browser.js?v=' . $browserJsHash;
```

Comment:
Was the path still correct in the line calculating the `$browserJsHash`?

Reply:
Yeah, it's a bit confusing because the path for the controller is different than the path that the browser sees.